src/napari/__main__.py

```diff
@@ -329,6 +329,15 @@ def _run() -> None:
         # it will collect it and hang napari at start time.
         # in a way that is machine, os, time (and likely weather dependant).
         viewer = Viewer()
+        script_path = get_settings().application.startup_script
+        if (
+            script_path
+            and os.path.exists(script_path)
+            and os.path.isfile(script_path)
+        ):
+            from napari_builtins.io._read import load_and_execute_python_code
+
+            load_and_execute_python_code(script_path)
```

Comment:
```suggestion
        if script_path:
            if os.path.exists(script_path) and os.path.isfile(script_path):
                from napari_builtins.io._read import load_and_execute_python_code
                load_and_execute_python_code(script_path)
            else:
                warnings.warn(
                    trans._(
                        "Startup script path is set to {script_path}, but this file does not exist or is not a file--ignoring.",
                        deferred=True,
                        script_path=script_path
                    )
                )
```

Let's warn the user if they set something and it doesn't work and we just ignore it.